solarforecastarbiter/tests/test_datamodel.py

```diff
@@ -442,14 +442,38 @@ def test___check_categories__():
 
 
 @pytest.mark.parametrize('metrics', [
-    (datamodel.ALLOWED_DETERMINISTIC_METRICS),
-    pytest.param(datamodel.ALLOWED_PROBABILISTIC_METRICS,
-                 marks=pytest.mark.xfail(raises=ValueError)),
+    (['rmse']),
+    (list(datamodel.ALLOWED_DETERMINISTIC_METRICS.keys())),
+    pytest.param(
+        ["bss"],
+        marks=pytest.mark.xfail(raises=ValueError, strict=True)
+    ),
+    pytest.param(
+        list(datamodel.ALLOWED_PROBABILISTIC_METRICS.keys()),
+        marks=pytest.mark.xfail(raises=ValueError, strict=True)
+    ),
 ])
 def test___check_metrics__(metrics, single_forecast):
     datamodel.__check_metrics__(single_forecast, metrics)
 
 
+@pytest.mark.parametrize('metrics', [
+    (['crps']),
+    (list(datamodel.ALLOWED_PROBABILISTIC_METRICS.keys())),
+    pytest.param(
+        ('rmse'),
+        marks=pytest.mark.xfail(raises=ValueError, stric=True)
+    ),
+    pytest.param(
+        list(datamodel.ALLOWED_DETERMINISTIC_METRICS.keys()),
+        marks=pytest.mark.xfail(raises=ValueError, stric=True)
+    ),
+])
+def test___check_metrics__probabilistic(metrics, prob_forecast_constant_value):
+    print(type(prob_forecast_constant_value))
```

Comment:
no print

Reply:
Thanks for catching that. I'll remove it.

Reply:
Corrected.